script/the_stack.md

```diff
@@ -0,0 +1,9 @@
+# The Stack
```

Comment:
Great page. I've been meaning to add a page on the [alternate stack](https://github.com/thunderbiscuit/opcode-explained/issues/1), and clearly a page on the stack itself should be there.